src/main/kotlin/com/greenbird/utilihive/stubs/sftp/SftpServerTestContext.kt

```diff
@@ -0,0 +1,323 @@
+package com.greenbird.utilihive.stubs.sftp
+
+import com.github.marschall.memoryfilesystem.MemoryFileSystemBuilder
+import org.apache.sshd.common.file.FileSystemFactory
+import org.apache.sshd.common.session.SessionContext
+import org.apache.sshd.server.SshServer
+import org.apache.sshd.server.auth.password.PasswordAuthenticator
+import org.apache.sshd.server.keyprovider.SimpleGeneratorHostKeyProvider
+import org.apache.sshd.server.session.ServerSession
+import org.apache.sshd.sftp.server.SftpSubsystemFactory
+import java.io.Closeable
+import java.io.IOException
+import java.io.InputStream
+import java.nio.charset.Charset
+import java.nio.file.FileStore
+import java.nio.file.FileSystem
+import java.nio.file.FileVisitResult
+import java.nio.file.Files
+import java.nio.file.Path
+import java.nio.file.PathMatcher
+import java.nio.file.SimpleFileVisitor
+import java.nio.file.WatchService
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.attribute.UserPrincipalLookupService
+import java.nio.file.spi.FileSystemProvider
+import java.util.*
+
+@Suppress("TooManyFunctions")
+class SftpServerTestContext
```

Comment:
The KDoc is currently inconsistent in its wording tense. Use the imperative, present tense consistenly. I.e. not "Returns the port" but "Return the port".

Reply:
Fixed